modules/monitoring-determining-why-prometheus-is-consuming-disk-space.adoc

```diff
@@ -6,19 +6,23 @@
 [id="determining-why-prometheus-is-consuming-disk-space_{context}"]
 = Determining why Prometheus is consuming a lot of disk space
 
-Labels are used to define attributes for metrics in the form of key-value pairs. A unique time series is created for every key-value pair. Adding many values to labels can result in an exponential increase in the number of time series created. This can impact Prometheus performance and can consume a lot of disk space.
+Developers can create labels to define attributes for metrics in the form of key-value pairs. The number of potential key-value pairs corresponds to the number of possible values for an attribute. An attribute that has an unlimited number of potential values is called an unbound attribute. For example, a `Name` attribute is unbound because it has an infinite number of possible name values.
```

Comment:
(nit) `customer_id` rather than `name` might be a better hint?

Reply:
@simonpasquier thank you for your review and for highlighting this. I have now replaced `Name` with `customer_id` in both module files. Thanks!